torch/csrc/jit/tracer.h

```diff
@@ -308,20 +308,22 @@ TORCH_API void ensureUniqueIfOutOfPlaced(
     const char* name,
     const at::Tensor& tensor);
 
+TORCH_API void addOutput(Node* node, const at::Tensor& tensor);
+TORCH_API void setOutput(Value* value, const at::Tensor& output);
+TORCH_API void addOutput(Node* node, const std::vector<at::Tensor>& list);
+TORCH_API void addOutput(Node* node, const c10::List<at::Tensor>& list);
```

Comment:
std::vector<at::Tensor>& list and `c10::List<at::Tensor>& list` seems duplicate

Reply:
Actually std::vector is unnecessary here, but without c10::List it will not work. I don't think there is any auto conversion between them.